src/custom/state/orders/updater.ts

```diff
@@ -312,7 +326,8 @@ export function ExpiredOrdersWatcher(): null {
         // validTo is either a Date or unix timestamp in seconds
         const validTo = typeof order.validTo === 'number' ? new Date(order.validTo * 1000) : order.validTo
 
-        return validTo < now
+        // let's get the current date, with our expired order validTo given a buffer time
```

Comment:
Not from this PR, but we should move this CHECK_EXPIRED_ORDERS_INTERVAL to our constant file too

Reply:
:+1: